packages/server/graphql/mutations/helpers/generateGroupSummaries.ts

```diff
@@ -0,0 +1,33 @@
+import getRethink from '../../../database/rethinkDriver'
+import OpenAIServerManager from '../../../utils/OpenAIServerManager'
+import {DataLoaderWorker} from '../../graphql'
+
+const generateGroupSummaries = async (
+  meetingId: string,
+  dataLoader: DataLoaderWorker,
+  facilitatorUserId: string
+) => {
+  const [reflections, reflectionGroups, facilitator] = await Promise.all([
+    dataLoader.get('retroReflectionsByMeetingId').load(meetingId),
+    dataLoader.get('retroReflectionGroupsByMeetingId').load(meetingId),
+    dataLoader.get('users').loadNonNull(facilitatorUserId)
+  ])
+  if (!facilitator.featureFlags.includes('aiSummary')) return
+  const r = await getRethink()
+  const manager = new OpenAIServerManager()
+  for (const group of reflectionGroups) {
+    const reflectionsByGroupId = reflections.filter(
+      ({reflectionGroupId}) => reflectionGroupId === group.id
+    )
+    if (reflectionsByGroupId.length <= 1) continue
+    const reflectionTextByGroupId = reflectionsByGroupId.map(
+      ({plaintextContent}) => plaintextContent
+    )
+    if (reflectionTextByGroupId.length === 0) continue
```

Comment:
-1 This cannot happen because `.map` does not change the length of the array and `reflectionsByGroupId` has length >= 2.